production_configs/20220303_tuned_ze40_south/lstmcpipe_config.yml

```diff
@@ -0,0 +1,47 @@
+## MC PRODUCTION FULL PIPELINE CONFIGURATION ##
+#
+# Template example to request a MC prod.
+
+
+workflow_kind: lstchain  # Not expected to be changed by the user
+prod_id: tuned_ze40_south        # ex: local_no_n_islands. Default; v00 (if key left empty or None)
```

Comment:
You can edit the prod_id like https://github.com/cta-observatory/lstmcpipe/blob/7cca15cce294362940c4f1eb4bc0b5da467f8ca8/production_configs/psf_tuned_zen20/lstmcpipe_config.yml#L7 to something like `zen40az180_dl1ab_tuned_psf_nsb`